src/app/components/content-finder-page/content-finder-page.component.ts

```diff
@@ -142,26 +144,40 @@ export class ContentFinderPageComponent implements OnInit {
          
          //populate a filter object with the params we just extracted
          let filter: Filter = new Filter(
-            title, format, moduleIdNumbers
+            title, formats, moduleIdNumbers
          );
 
          this.sendSearch(filter);
       }
    }
 
+   toggleFormat(format : string){
```

Comment:
What is this method doing? Can you create a JSDoc Comment here similar to the submit method?

Reply:
Can do

Reply:
Change pushed

Reply:
These types of comments should be JSDoc comments. They are structured as a multiline comment, except with 2 asterisks at the top. Like this:

/**
 *
 *
 */

This type of comment allows IDEs to display the comment as a hover tooltip. Much preferable to Standard // comments.

Reply:
Copy, fixed.